src/views/services-details/ServicesDetails.vue

```diff
@@ -50,6 +50,10 @@
       services() {
         return this.getServicesByType;
       },
+
+      banners() {
+        return this.getServicesByType[0];
```

Comment:
What if getServicesByType is null or empty?

Reply:
Refactored